js/components/frame.js

```diff
@@ -161,8 +195,10 @@ class Frame extends ImmutableComponent {
       this.webview.allowRunningPlugins = true
     }
 
-    if (!guestInstanceId || src !== 'about:blank') {
-      this.webview.setAttribute('src', isSourceAboutUrl(src) ? getTargetAboutUrl(src) : src)
+    if (!guestInstanceId || newSrc !== 'about:blank') {
+      // XXX: Should webview src always be set to location, not src? Location
```

Comment:
I think that is the case now, at some point in this function we couldn't do that but things have changed since then.